doc/static-badges.md

```diff
@@ -0,0 +1,12 @@
+# Static Badges
+
+It is possible to use shields.io to make a wide variety of badges displaying static text and/or logos. For example:
+
+- ![any text you like](https://img.shields.io/badge/any%20text-you%20like-blue) - https://img.shields.io/badge/any%20text-you%20like-blue
+- ![just the messgae](https://img.shields.io/badge/just%20the%20message-8A2BE2) - https://img.shields.io/badge/just%20the%20message-8A2BE2
+- ![for the badge style](https://img.shields.io/badge/for%20the%20badge%20style-20B2AA?style=for-the-badge) - https://img.shields.io/badge/for%20the%20badge%20style-20B2AA?style=for-the-badge
+- ![with a logo](https://img.shields.io/badge/with%20a%20logo-grey?style=for-the-badge&logo=javascript) - https://img.shields.io/badge/with%20a%20logo-grey?style=for-the-badge&logo=javascript
+
+Full documentation of styles and parameters: https://shields.io/#styles
+
+More documentation on logos: https://github.com/badges/shields/blob/master/doc/logos.md
```

Comment:
To https://github.com/badges/shields/blob/master/doc/logos.md we can add info about the change in the `logoColor` param introduced in this PR (https://github.com/badges/shields/pull/8263) and how to use it.

Reply:
d5e2bd3